lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -1,3 +1,5 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
```

Comment:
GPLv3 or Apache2? We're licensing charms using this library as Apache 2

Reply:
Er, absolultely. Bad copy/paste...